pkg/shell/cockpit-dashboard.js

```diff
@@ -410,7 +410,18 @@ PageDashboard.prototype = {
                 update_series();
             }
 
-            return render;
+            function throttled_render() {
+                var timer = null;
+                return function () {
+                    if (timer)
+                        window.clearTimeout(timer);
```

Comment:
Technically the set timer is the flag, so you would render too often, since we don't have real parallelism. The upside is simpler code, which isn't a bad thing.